### PR TITLE
feat: adjust styling add signup page

### PR DESCRIPTION
**Pull request by kittybest**
**Description**

- [x] adjust theme mode to "light"
- [x] adjust light theme color set
- [x] adjust footer and header
- [x] add a signup page

**Related Issue**
relate to #55

### CODE REVIEW

**Comment by kittybest** [comment]
Thinking it's ready to merge for the current logic.
If the signup flow or other details need to be modified, then should open another PR for that.

File: src/config.ts [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
   voteLimit: 50,
   startsAt: new Date(process.env.NEXT_PUBLIC_START_DATE!),
   registrationEndsAt: new Date(process.env.NEXT_PUBLIC_REGISTRATION_END_DATE!),
-  reviewEndsAt: new Date(process.env.NEXT_PUBLIC_REVIEW_END_DATE!),
   resultsAt: new Date(process.env.NEXT_PUBLIC_RESULTS_DATE!),
   skipApprovedVoterCheck: ["true", "1"].includes(
     process.env.NEXT_PUBLIC_SKIP_APPROVED_VOTER_CHECK!,
   ),
   tokenName: process.env.NEXT_PUBLIC_TOKEN_NAME!,
+  eventName: process.env.NEXT_PUBLIC_EVENT_NAME!,
```
Comment (ctrlc03) [inline src/config.ts:23]: let's put a default one of maci-rpgf?

File: src/components/EligibilityDialog.tsx [hunk 2]
```diff
@@ -1,38 +1,61 @@
...
     >
-      <div>
-        Only badgeholders are able to vote in {metadata.title}. You can find out
-        more about how badgeholders are selected{" "}
-        <Link href="" target="_blank" className="underline underline-offset-2">
-          here
-        </Link>
-        .
-      </div>
+      {isEligibleToVote ? (
```
Comment (ctrlc03) [inline src/components/EligibilityDialog.tsx:41]: I think this should also check if the user is already signed up or not

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Thanks left some comments

**Review by ctrlc03 — APPROVED** [review]

**Comment by crisgarner** [comment]
As we are approaching the main launch of TGU and MX rounds [can](url) we merge this into a development branch and only merge to main critical fixes?

File: src/pages/signup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        <h2 className="max-w-screen-lg text-center font-mono">
+          {config.roundId.toUpperCase()}
+        </h2>
+        <p className="max-w-screen-md text-center text-xl">
+          {config.startsAt.toLocaleDateString("en-US", {
+            year: "numeric",
+            month: "long",
+            day: "numeric",
+          })}{" "}
+          -{" "}
```
Comment (0xmad) [inline src/pages/signup/index.tsx:34]: Spacing should be done with styles

File: src/providers/index.tsx [hunk 4]
```diff
@@ -24,6 +25,24 @@ const getSiweMessageOptions: GetSiweMessageOptions = () => ({
   statement: process.env.NEXT_PUBLIC_SIGN_STATEMENT ?? "Sign in to OpenPGF",
 });
 
+const customTheme: Theme = {
+  blurs: {
+    ...lightTheme().blurs,
```
Comment (0xmad) [inline src/providers/index.tsx:30]: lightTheme() call can be moved and called only once

File: src/components/Info.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+import { EInfoCardState } from "~/utils/types";
+import { useMaci } from "~/contexts/Maci";
+import { config } from "~/config";
+
+export function Info() {
+  const { votingEndsAt } = useMaci();
+
+  const steps = [
+    {
+      label: "APPLICATION",
```
Comment (0xmad) [inline src/components/Info.tsx:13]: Same here with uppercase styles

File: src/components/Info.tsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    start.getMonth() === end.getMonth() &&
+    start.getFullYear() === end.getFullYear()
+  ) {
+    return `${start.getDate()} - ${end.getDate()} ${end.toLocaleDateString("en-US", { month: "short" })} ${end.getFullYear()}`;
+  } else if (start.getFullYear() === end.getFullYear()) {
+    return `${start.getDate()} ${start.toLocaleDateString("en-US", { month: "short" })} - ${end.getDate()} ${end.toLocaleDateString("en-US", { month: "short" })} ${end.getFullYear()}`;
+  } else {
+    return `${start.getDate()} ${start.toLocaleDateString("en-US", { month: "short" })} ${start.getFullYear()} - ${end.getDate()} ${end.toLocaleDateString("en-US", { month: "short" })} ${end.getFullYear()}`;
+  }
+}
```
Comment (0xmad) [inline src/components/Info.tsx:113]: For date and time format use this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

**Comment by kittybest** [comment]
> As we are approaching the main launch of TGU and MX rounds can we merge this into a development branch and only merge to main critical fixes?

yes we could, since I know eth MX has already built their own frontend?

File: src/pages/signup/index.tsx (hunk 3 above)
Comment (kittybest) [reply src/pages/signup/index.tsx]: how about now that using flex, span, and gap?

File: src/components/EligibilityDialog.tsx [hunk 7]
```diff
@@ -1,38 +1,87 @@
...
+              : "Disconnect"
+        }
+        buttonAction={
+          isRegistered
+            ? () => router.push("/projects")
+            : isEligibleToVote
+              ? handleSignup
+              : () => disconnect()
+        }
+      />
```
Comment (0xmad) [inline src/components/EligibilityDialog.tsx:84]: Nested `:?` is a bad practice

File: src/pages/signup/index.tsx (hunk 3 above)
Comment (0xmad) [reply src/pages/signup/index.tsx]: You can just use margin style prop

File: src/components/EligibilityDialog.tsx [hunk 8]
```diff
@@ -1,38 +1,92 @@
...
+  const [openDialog, setOpenDialog] = useState<boolean>(!!session?.publicKey);
+  const { onSignup, isEligibleToVote, isRegistered } = useMaci();
+  const router = useRouter();
 
-  if (isLoading || !address || !session || error) return null;
+  const onError = useCallback(() => toast.error("Signup error"), []);
+  const handleSignup = useCallback(async () => {
+    await onSignup(onError);
+    setOpenDialog(false);
+  }, [onSignup, onError]);
```
Comment (0xmad) [inline src/components/EligibilityDialog.tsx:22]: setOpenDialog is missing here

File: src/components/EligibilityDialog.tsx [hunk 9]
```diff
@@ -1,38 +1,92 @@
...
+
+  useEffect(() => {
+    setOpenDialog(!!session?.publicKey);
+  }, [session]);
```
Comment (0xmad) [inline src/components/EligibilityDialog.tsx:26]: setOpenDialog is missing here and it should be `session?.publicKey` dep

**Review by 0xmad — APPROVED** [review]
@kittybest thanks!

**Pull request merged**